api/app/responses/api.py

```diff
@@ -288,10 +288,15 @@ def get(self, event_id):
         include_unsubmitted = args['include_unsubmitted']
         question_ids = args['question_ids[]']
         language = args['language']
-        
+        current_user = user_repository.get_by_id(g.current_user['id'])
+
         print(('Include unsubmitted:', include_unsubmitted))
 
-        responses = response_repository.get_all_for_event(event_id, not include_unsubmitted)
+        if current_user.is_event_admin(event_id):
+            responses = response_repository.get_all_for_event(event_id, not include_unsubmitted)
+        
+        elif current_user.is_action_editor(event_id):
```

Comment:
I wonder if there's a better name than "action-editor" that's not a journal-specific term, since the functionality is not limited to a journal event type. Will try think of something too!